change/@fluentui-web-components-a2072109-cb0f-486e-9cbe-81ebc19d21c4.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentui-web-components-v3 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33084&commitId=34b22b43ca1e5c6398f8e6baa30e8d96ee191efd&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar.Image.chromium.png|284| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>